### DISCUSSION TITLE
Is there a cleaner way to do a join without having to remove the duplicated column?

### DISCUSSION THREAD

**Discussion by kesmit13** [ibis-project/ibis#4606] [Q&A] [answered]
I'm just trying to clean this up by not having to specify suffixes and drop the extra column in code like this.
```
freq.join(gender, freq.Sex == gender.Sex, suffixes=('', '_')).drop('Sex_')
```

**Comment by cpcloud** [comment]
Yeah, we could make this more convenient for the (probably very common) case of inner join with only equality predicates, see [here](https://github.com/ibis-project/ibis/issues/1223#issuecomment-345476096) for the rationale.

**Comment by kesmit13** [comment]
After thinking about it a while, I did realize that this was a very special case. It would be nice if it did the "expected" thing though if possible.

**Reply by cpcloud** [reply]
It is a special case, but it's a common one so I think we're definitely interested in making the UX here better.

**Comment by cpcloud** [comment]
There is now a cleaner way to do this arriving in 6.0 thanks to @jcrist in #5883.

**Comment by cpcloud** [comment, marked as answer]
This has been fully implemented. Inner joins with equality-predicates-only will not duplicate columns.
